pages/contactus.tsx

```diff
@@ -9,24 +9,68 @@ import TextField from "snet-ui/TextField";
 const categories = ["Airdrop Enquiry"];
 
 export default function ContactUs() {
+  const [message, setMessage] = useState("");
+  const [email, setEmail] = useState("");
+
+  const [error, setError] = useState({ email: "", message: "" });
   const [category, setCategory] = useState("Airdrop Enquiry");
 
   const handleChange = (event: SelectChangeEvent) => {
     setCategory(event.target.value as string);
   };
+  const validateEmail = () => {
```

Comment:
Always name the functions descriptively and appropriately. 

in this function, you are handling the value change of email. You are not validating it. 

So an appropriate name would be `handleEmailChange`